collect_app/src/main/AndroidManifest.xml

```diff
@@ -146,16 +146,11 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".activities.FormManagerList"
-            android:label="@string/app_name" />
-        <activity
             android:name=".activities.FormDownloadList"
             android:label="@string/app_name" />
         <activity
-            android:name=".activities.DataManagerList"
-            android:label="@string/app_name" />
-        <activity
             android:name=".activities.FileManagerTabs"
+            android:configChanges="orientation|screenSize"
```

Comment:
Without this line, the current position and state of the listview is reset. All the sorting performed will also be forgotten. Also the state of the select and delete button would be lost as well. For testing you can sort the list and select all items. The label on button will change to `clear all`. Now rotate the screen and the label will go back to `select all` and the sorting would be reverted back to default ordering